index.html

```diff
@@ -1135,13 +1157,19 @@ <h2>Security Considerations</h2>
             not understanding the implications of this section can result in
             security vulnerabilities.
          </p>
+         <p>
+            When using the <code><a href="#jsonschemacredential">JsonSchemaCredential</a></code> 
+            <code>type</code> implementers are advised to review the <a data-cite="VC-DATA-MODEL-2.0/#security-considerations">
+            Security Considerations</a> outlined in the [[VC-DATA-MODEL-2.0]].
+         </p>
          <section class="informative">
             <h3>Issuer Impersonation</h3>
             <p>
                It is possible for a schema to become authoritative, such as schemas
                provided by a recognized industry group like a consoritum of financial
                companies. To avoid confusion as to the authorship of <a>credential schemas</a>
```

Comment:
```suggestion
               It is possible for a schema to become authoritative, such as a schema
               provided by a recognized industry group like a consortium of financial
               companies. To avoid confusion as to the authorship of <a>credential schemas</a>,
```